rules/qc.smk

```diff
@@ -1,29 +1,33 @@
 ## RSEQC
+
 rule rseqc_gtf2bed:
     input:
         config["ref"]["annotation"]
     output:
-        "qc/rseqc/annotation.bed"
+        bed="qc/rseqc/annotation.bed",
+        db=temp("qc/rseqc/annotation.db")
     log:
-        "logs/gtf2bed.log"
+        "logs/rseqc_gtf2bed.log"
+    conda:
+        "../envs/gffutils.yaml"
     params:
-        script = "../scripts/gtf2bed.pl"
+        script="../scripts/gtf2bed.py"
     shell:
```

Comment:
Here, you can make use of the script directive:
```
script:
    "../scripts/gtf2bed.py"
```

Then, the script becomes much shorter, because you can omit all the boilerplate. Example: https://github.com/snakemake-workflows/rna-seq-star-deseq2/blob/master/scripts/count-matrix.py

Reply:
yes can be done and I will do it if you want it. However, I personally think that this is not good practice it makes it much more of a black box and understanding what is happening is much more difficult. Also, it makes the script really only usable for this workflow. 

Anyway, say the word and I will do it.
Seb

Reply:
ok I did it. Question, will the log in the rule rseqc_gtf2bed do anything now?

Reply:
Indeed, if the script is of use in other contexts, then it makes sense to have a full CLI. However, if that is the case, it should rather not be living in this workflow, but be a proper python package available via pypi and conda. So, for something in here, it greatly increases readability and accessibility in the "without-boilerplate" form.

Reply:
Actually, I would at some point move all the rseqc rules into Snakemake wrappers. Then, the gtf2bed thing could become a wrapper as well. But for now, the current version is perfect.